.github/workflows/health_check.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - name: Notify slack
         uses: kpritam/slack-job-status-action@v1
         with:
+          if: ${{ failure() }}
```

Comment:
@kleinjm I'm pretty sure that this will work fine with this and notify us of failure.

I would rather we trust the system and be notified only of failures in order to diminish noise.

Reply:
absolutely. And FWIW, that artifact upload GH action uses `failure()` and works as expected (doesn't try to upload on success case) so this should work